status-im/status-desktop
# feat(@desktop/onbiarding): Show Allow Notification page during first run

Issue #5583

### What does the PR do

During the first run Allow Notification page is shown.
The page is only visible for Mac system.

Design: https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1075%3A312335

Testing steps:
1. Remove Status directory (create a backup before)
2. Run application
3. The 1st page should be "Allow Notification" page

### Affected areas

Onboarding

### Screenshot of functionality

![image](https://user-images.githubusercontent.com/61889657/169579778-4718c025-2cef-437c-8699-b17eb20a0a9e.png)
